src/components/tokeninfo.js

```diff
@@ -0,0 +1,14 @@
+class TokenInfo {
+  constructor() {
+    this.ref = document.querySelector(".tokeninfo");
+  }
+  render(props) {
+    if (props.token) {
+      const input = document.createElement("input");
+      input.setAttribute("type", "text");
+      document.body.appendChild(input);
```

Comment:
it should be liek this, right?
```suggestion
      this.ref.appendChild(input);
```